main.js

```diff
@@ -3893,6 +3908,24 @@ function init(compactGroupId) {
         logger.info(hostLogPrefix + ' ' + tools.appName + '.js-controller version ' + version + ' ' + ioPackage.common.name + ' starting');
     }
 
+    const pluginSettings = {
+        scope: 'controller',
+        namespace: hostObjectPrefix,
+        logNamespace: hostLogPrefix,
+        log: logger,
+        iobrokerConfig: config,
+        parentPackage: null
+    };
+    try {
+        pluginSettings.parentPackage = JSON.parse(fs.readFileSync(__dirname + '/package.json', 'utf8'));
+    }
+    catch (err) {
+        logger.error(hostLogPrefix + ' Can not read js-controller package.json');
+    }
+    pluginHandler = new PluginHandler(pluginSettings);
+    pluginHandler.addPlugins(ioPackage.common.plugins, __dirname); // Plugins from io-package have priority over ...
+    pluginHandler.addPlugins(config.plugins, __dirname);           // ... plugins from iobroker.json
```

Comment:
Does `addPlugins` override plugins or just add them? If it's supposed to override, you need to reverse these lines because otherwise io-package plugins don't have priority.

Reply:
No it will just add and "already existing" once are ignored when trying to be added again

Reply:
OK